### PR TITLE
einar/Change expand icon to minimize icon when archive accordion open

### LINKED ISSUE #147: Change expand icon to minimize icon when archive accordion open
When you expand a workshop series in the workshop archive, the + remains a +. Want the + to be a - when the accordion is open to indicate it will minimize the section.

### PR DESCRIPTION
**Pull request by EinarBalan**
Fixes #147

### CODE REVIEW

File: src/components/ArchivePage/EventInfoItem.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import { useState } from 'react';
```
Comment (TimothyGu) [inline src/components/ArchivePage/EventInfoItem.js:2]: nit:
```suggestion
import React, { useState } from 'react';
```

File: src/components/ArchivePage/EventInfoItem.js [hunk 2]
```diff
@@ -45,9 +47,11 @@ const useStyles = makeStyles(theme => ({
 
 function EventInfoItem({ name, mainLink, tags, director, workshops }) {
 	const classes = useStyles();
-	return <Accordion classes={{ root: classes.paperRoot }}>
+	const [isExpanded, setExpanded] = useState(true);
+	return <Accordion classes={{ root: classes.paperRoot }}
+				onChange={() => setExpanded(isExpanded => !isExpanded)}>
```
Comment (TimothyGu) [inline src/components/ArchivePage/EventInfoItem.js:52]: nit:

```suggestion
	return <Accordion classes={{ root: classes.paperRoot }}
	                  onChange={() => setExpanded(isExpanded => !isExpanded)}>
```

Use spaces for indentations beyond the `return` keyword, as the size of tabs can change based on the editor:

image

**Review by TimothyGu — APPROVED** [review]

File: src/components/ArchivePage/EventInfoItem.js (hunk 2 above)
Comment (EinarBalan) [reply src/components/ArchivePage/EventInfoItem.js]: linter error :/ <br>
Screen Shot 2021-01-17 at 9 26 16 PM

Comment (TimothyGu) [reply src/components/ArchivePage/EventInfoItem.js]: Bah

Comment (TimothyGu) [reply src/components/ArchivePage/EventInfoItem.js]: The original is fine then :)

**Pull request merged**